applications/projects/CMakeLists.txt

```diff
@@ -7,9 +7,7 @@ if(NOT Qt5Core_FOUND)
     set(SOFA_HAVE_QT_LIBRARIES 0)
 endif()
 
-if(SOFA_HAVE_QT_LIBRARIES)
-    sofa_add_application(Modeler Modeler OFF)
-endif()
```

Comment:
Why? I would have kept the condition.

Reply:
I did too at first, but it's a bit weird to do a check required for activation to then deactivate the thing. If the modeler require qt, then it should check for qt itself.

Reply:
> If the modeler require qt, then it should check for qt itself.

I agree with that but you can't just remove the condition then, you should move it to Modeler's CMakeLists (and fail or warn+return if false).

Reply:
Modeler does `find_package(Qt5 COMPONENTS Core QUIET)`
Maybe just change that to `REQUIRED` ?

Reply:
I'm good with that :+1: